modules/api_test/tests/src/Functional/ApiClient.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\Tests\api_test\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * @group headless
+ * @group api_test
+ */
+class ApiClient extends BrowserTestBase {
```

Comment:
The class name should probably end with Test. Also, it just tests that a token is acquired, so can it be ApiTokenTest or something more descriptive?